src/wasm/wasm-s-parser.cpp

```diff
@@ -1543,7 +1520,7 @@ Index SExpressionWasmBuilder::parseMemoryLimits(Element& s, Index i) {
 void SExpressionWasmBuilder::parseMemory(Element& s, bool preParseImport) {
   if (wasm.memory.exists) throw ParseException("too many memories");
   wasm.memory.exists = true;
-  wasm.memory.imported = preParseImport;
+  // FIXME wasm.memory.imported = preParseImport;
```

Comment:
Is this handled further down, or does this still need a fix?

Reply:
I can just remove those outdated comments - we no longer need to mark "imported", as Importables track that in a better way anyhow.

Reply:
That's what I was thinking yeah, sgtm